### DISCUSSION TITLE
Can't upgrade to pro! "The plan change was successful.... will be updated to the free monthly plan.

### DISCUSSION THREAD

**Discussion by memoakten** [community/community#140224] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

- I'm currently on the free plan. I want to upgrade to pro.
- I goto https://github.com/settings/billing/plans and click on `upgrade to pro`
- enter all my correct details, select `yearly: $48.00 / year` and click on `upgrade`
- it goes back to the billing page. I'm still on the free plan and the message says:
`The plan change was successful. @memo will be updated to the free monthly plan`.

**Comment by ckreutz** [comment]
Same for me

**Comment by mikk1max** [comment]
### 1. Clear Cache and Cookies:
Sometimes, browser caching or cookies can interfere with the functionality of web pages. Clear your browser's cache and cookies or try accessing the GitHub billing page in incognito mode or a different browser.

### 2. Check Email Confirmation:
Look in your email inbox for any confirmation messages from GitHub. Sometimes the upgrade may take a few minutes to process, and you'll receive a confirmation email once the payment is successful.

### 3. Check GitHub's Status Page:
Occasionally, GitHub might experience issues with their billing or payment systems. You can check the GitHub Status Page to see if there are any ongoing issues.

### 4. Contact GitHub Support:
Since you're getting the message "The plan change was successful... will be updated to the free monthly plan", it sounds like a potential glitch. You can reach out to GitHub Support directly by submitting a request through their [Contact Support](https://support.github.com/contact) page.
Provide them with details of the issue, including any screenshots if possible. Make sure to include the exact text of the message you received.
